src/cart/crt_swim.c

```diff
@@ -544,16 +632,40 @@ int crt_swim_init(int crt_ctx_idx)
 
 	rc = crt_proto_register(&crt_swim_proto_fmt);
 	if (rc) {
-		D_ERROR("crt_proto_register() failed=%d\n", rc);
+		D_ERROR("crt_proto_register() failed "DF_RC"\n", DP_RC(rc));
 		D_GOTO(cleanup, rc);
 	}
 
 	rc = crt_register_progress_cb(crt_swim_progress_cb, crt_ctx_idx, NULL);
 	if (rc) {
-		D_ERROR("crt_register_progress_cb() failed=%d\n", rc);
+		D_ERROR("crt_register_progress_cb() failed "DF_RC"\n",
+			DP_RC(rc));
 		D_GOTO(cleanup, rc);
 	}
 
+	if (!d_fault_inject_is_enabled())
+		D_GOTO(out, rc);
+
+	crt_swim_should_fail = false; /* disabled by default */
+	crt_swim_fail_hlc = 0;
+	crt_swim_fail_delay = 10;
+	crt_swim_fail_id = SWIM_ID_INVALID;
+
+	/* Search the attr in inject yml first */
+	d_fa_swim_drop_rpc = d_fault_attr_lookup(CRT_SWIM_FAIL_DROP_RPC);
+	if (d_fa_swim_drop_rpc != NULL) {
+		D_ERROR("*** fa_swim_drop_rpc: id=%u/0x%x, "
```

Comment:
This doesn't need to be D_ERROR

Reply:
I definitely would like to see this message even if user specify ERR debug level. This will highlight issues we will have. I should understand why SWIM issue happens.

Reply:
This is only emitted if FI is enabled?  There is the severity and facility to search for so it certainly doesn't need the *** part.

Reply:
Yes, this message will appears only when FI is enabled. For regular code path it never appears.

Reply:
I tend to agree on not adding the ***.   There should be a facility or some string to search instead.   If you added "IMPORTANT" or "ALWAYS" type, that would suffice as well.